src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -0,0 +1,14 @@
+databaseChangeLog:
+  - changeSet:
```

Comment:
what's that?

Reply:
in previous tasks it was necessary to add Liquibase

Reply:
You should add the tables for the entity, `books`, and other,
but not test_table